src/.env.sample

```
@@ -0,0 +1,259 @@
+
+# Scheduler service base url
+SCHEDULER_SERVICE_BASE_URL=/scheduler/
+
+# Default organization code
+DEFAULT_ORGANIZATION_CODE=default_code
+
+# Allowed host by -CORS security
+ALLOWED_HOST=http://exampleDomain.com
+
+#Authentication meathod to be used
+AUTH_METHOD=jwt_only #or user_service_authenticated
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)